templates/github/.github/workflows/changelog.yml.j2

```diff
@@ -0,0 +1,45 @@
+{% include 'header.j2' %}
+{% from 'macros.j2' import
+  checkout,
+  setup_python,
+  set_secrets,
+  install_python_deps,
+with context %}
+---
+name: {{ plugin_camel_short | default("Pulp") }} changelog update
+on:
+  push:
+    branches:
+      - {{ plugin_default_branch }}
+    paths:
+      - CHANGES.rst
+      - CHANGES.md
```

Comment:
I think it should be:
```suggestion
      - CHANGES/**
```

Reply:
I don't think so. We want to republish the generated changelog (either rst or md) and not every time something happened with the "unreleased" snippet files.

Reply:
Oh! I misunderstood the PR, I thought it was for building changelogs whenever a new entry was added.

Reply:
No, it's publishing the complete changelog whenever it was changed.